packages/netlify-cms-core/src/components/MediaLibrary/MediaLibraryTop.js

```diff
@@ -101,6 +104,11 @@ function MediaLibraryTop({
           placeholder={t('mediaLibrary.mediaLibraryModal.search')}
           disabled={searchDisabled}
         />
+        <MediaLibraryCreateFolder
+          onKeyDown={onCreateFolder}
+          placeholder='Create folder' // TODO multilingual
```

Comment:
To allow translations you just have to use `t` and add the default value in the english index file. You can see an example in the PR #4938 :wink:
Based on the translation line 104, I think we should name this one this way:
```suggestion
          placeholder={t('mediaLibrary.mediaLibraryModal.createFolder')}
```